.github/test-spec.yml

```diff
@@ -297,14 +297,19 @@
 "CI-matter-test":
   - "drivers/mpsl/**/*"
   - "drivers/hw_cc310/*"
-  - "drivers/net/*"
+  - "drivers/entropy/*"
+  - "drivers/wifi/**/*"
   - "subsys/mpsl/**/*"
   - "subsys/ieee802154/**/*"
   - "subsys/bootloader/**/*"
   - "subsys/partition_manager/**/*"
   - "subsys/nfc/**/*"
   - "subsys/nrf_rpc/**/*"
-  - "subsys/net/**/*"
+  - "subsys/dfu/**/*"
+  - "subsys/fw_info/*"
+  - "subsys/pcd/**/*"
+  - "subsys/bluetooth/controller/*"
+  - "subsys/net/openthread/*"
```

Comment:
Looks like we miss `subsys/net/l2/wifi` - this is related to Wi-Fi connection management and Wi-Fi shell support.

Reply:
I think you're looking at Zephyr directory. In this file there are only nrf-sdk related triggers, and I can't see subsys/net/l2 there.

Reply:
Sorry, I was looking at the 2.1.1 revision of NCS, indeed the l2/wifi has been removed recently:
https://github.com/nrfconnect/sdk-nrf/commit/2d700b9a1b4e812de0c4210f9c77b1647d326947